portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,28 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>hey it's malachi!</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+  <style>
+      img {
+  width:360px;
+  height:640px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+body {background-color: #D4E7FF;}
+</style>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+<h2><center>★ MALACHI BLENMAN'S PORTFOLIO ★</center></h2>
```

Comment:
FYI (It's fine to leave for this PR if you want). The `<center>` tag is deprecated and not really recommended anymore. The current approach is to use CSS. See this article: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center

It's fine to leave for now, and all browsers still support it. Just thought I'd let you know.